README.md

```diff
@@ -49,7 +49,7 @@ If you are new to Graviton and want to understand how to identify target workloa
 |Interconnect	|CMN-600	|CMN-650	|
 |Architecture revision	|ARMv8.2-a	|ARMv8.4-a	|
 |Additional  features	|fp16, rcpc, dotprod, crypto	|sve, rng, bf16, int8, crypto	|
-|Recommended -mcpu flag	|neoverse-n1	|neoverse-512tvb	|
+|Recommended `-mcpu` flag	([more information](c-c++#enabling-arm-architecture-specific-features))|`neoverse-n1` | `neoverse-512tvb` |
```

Comment:
Broken link, I think you need `c-c++.md#...`

I'm fine with this formatting as-is, but an alternative is to put two values in the table data: "performance: xxx, compatibility: yyy"

Reply:
I was sure I already tested that link! 😞 

I wanted to avoid making it too complex in the top level guidance. I thought adding the detail here would potentially confusing.